src/clj/game/cards/assets.clj

```diff
@@ -128,6 +128,19 @@
                  :effect (effect (trash card))
                  :msg (msg "swap " (:advance-counter card 0) " cards in HQ and Archives")}]}
 
+   "Amani Senai"
+   (let [get-last-stolen-pts (fn [state] (:agendapoints (last (get-in @state [:runner :scored]))))
+         get-last-scored-pts (fn [state] (:agendapoints (last (get-in @state [:corp :scored]))))
+         senai-ability (fn [trace-base-func]
+                          {:optional {:prompt "Trace with Amani Senai?" :player :corp
```

Comment:
This is indented one space too far. Not huge, but let's be consistent. Let's also put the `:player :corp` on its own line for readability.